documentation/src/main/asciidoc/ch02.asciidoc

```diff
@@ -95,7 +95,7 @@ It is possible to specify constraints directly on the type argument of a
 parameterized type: these constraints are called container element constraints.
 
 This requires that `ElementType.TYPE_USE` is specified via `@Target`
-in the constraint definition. As of Jakarta Bean Validation 2.0, built-in Jakarta Bean Validation as well as
+in the constraint definition. As of Jakarta Bean Validation 2.0, built-in Jakarta Validation as well as
```

Comment:
```suggestion
in the constraint definition. As of Jakarta Validation 2.0, built-in Jakarta Validation as well as
```

And is the version correct?

Reply:
yeah I was a bit puzzled about how to change this 😄 
I mean the `ElementType.TYPE_USE` is available since Java Bean Validation 2.0 (the spec that was not yet Jakarta). But since we are talking about `Jakarta Bean Validation 2.0` here .. which was a rebrand-to-jakarta-only thing... should we change the names of "older" specs or is the shorter name only applicable to 3.1 and forward? 😖😖😖 🤔 😃

Reply:
No you're right, I didn't pay enough attention. Please ignore :)